lib/generate-svg.js

```diff
@@ -307,5 +307,10 @@ module.exports = (sources = [], options = {}, warnings = []) => {
         svg.setAttribute('height', (sizes.height.reduce((a, b) => a + b, 0) + ((sizes.height.length - 1) * options.sprite.gutter)).toString());
     }
 
+    // Add custom attributes to spritemap
+    Object.entries(options.output.svg.attributes || {}).forEach(([name, value]) => {
```

Comment:
I don't think the `{}` is required here? It defaults to `{}` in `options-formatter.js` and `options-validator.js` ensures it's an object.

Reply:
I think that's correct, but the tests don't run the options through `options-formatter.js`, so a number of the existing tests were failing without this. Would you prefer that I remove this and ensure `options.output.svg.attributes` is defined for all the tests?

Reply:
Yeah, that'd be great. Maybe it's also a good idea to update all tests that contain an options object to pass them through the formatter in the test... I'm definitely in favor of not "bloating" the main code just to satisfy tests.

Reply:
@cascornelissen Updated. Thoughts?